django_app/redbox_app/redbox_core/views/chat_views.py

```diff
@@ -54,11 +54,11 @@ def get(self, request: HttpRequest, chat_id: uuid.UUID | None = None) -> HttpRes
             "chat_title_length": settings.CHAT_TITLE_LENGTH,
             "llm_options": [
                 {
-                    "name": llm,
-                    "default": llm == AbstractAISettings.ChatBackend.GPT_4_OMNI.value,
+                    "name": llm.name,
+                    "default": llm.is_default,
                     "selected": llm == chat_backend,
                 }
-                for _, llm in AbstractAISettings.ChatBackend.choices
+                for llm in ChatLLMBackend.objects.all()
```

Comment:
filter on enabled now as it is a flag?

Reply:
good point!

Reply:
[3b62d91](https://github.com/i-dot-ai/redbox/pull/1084/commits/3b62d91ed62d14da58934385cdc85ec3b47c1add)